net-p2p/qbittorrent/qbittorrent-9999.ebuild

```diff
@@ -56,28 +56,22 @@ BDEPEND+="
 
 DOCS=( AUTHORS Changelog {CONTRIBUTING,README}.md )
 
-src_prepare() {
-	MULTIBUILD_VARIANTS=()
-	use gui && MULTIBUILD_VARIANTS+=( gui )
-	use webui && MULTIBUILD_VARIANTS+=( nogui )
-
-	cmake_src_prepare
-}
-
 src_configure() {
+	MULTIBUILD_VARIANTS+=( $(usev gui) $(usev webui nogui) )
+
 	my_src_configure() {
 		local mycmakeargs=(
-			# musl lacks execinfo.h
-			-DSTACKTRACE=$(usex !elibc_musl)
-			# More verbose build logs are preferable for bug reports
-			-DVERBOSE_CONFIGURE=ON
-			-DWEBUI=$(usex webui)
+			-DVERBOSE_CONFIGURE=ON # for bug reports
+			-DSTACKTRACE=$(usex !elibc_musl) # musl lacks execinfo.h
 			-DTESTING=$(usex test)
+			-DWEBUI=$(usex webui)
 		)
 
+		# upstream supports building just gui or nogui
+		# https://github.com/qbittorrent/qBittorrent/blob/master/CMakeLists.txt
+		# so we build the project twice (see #839531 for details)
+		# Fedora does the same: https://src.fedoraproject.org/rpms/qbittorrent
```

Comment:
While I do appreciate the better explanation overall, I don't think that linking the CMakeLists.txt conveys anything. It's not even the one where the gui/nox targets are defined...